inc/credential.class.php

```diff
@@ -345,21 +345,15 @@ public static function dropdownCredentials($params = [])
     {
         global $CFG_GLPI;
 
-        $p = [];
-        if ($params['id'] == -1) {
-            $p['value']    = '';
-            $p['itemtype'] = '';
-            $p['id']       = 0;
-        } else {
-            $credential = new PluginGlpiinventoryCredential();
-            $credential->getFromDB($params['id']);
-            if ($credential->getFromDB($params['id'])) {
-                $p = $credential->fields;
-            } else {
-                $p['value']    = '';
-                $p['itemtype'] = '';
-                $p['id']       = 0;
-            }
+        $p = [
+            'value'    => '',
+            'itemtype' => '',
+            'id'       => 0
+        ];
+
+        $credential = new PluginGlpiinventoryCredential();
+        if ($credential->getFromDB($params['id'])) {
```

Comment:
```suggestion
        if ($params['id'] > 0 && $credential->getFromDB($params['id'])) {
```

Reply:
won't `getFromDB` return false if -1?

Reply:
Yes, but it will try to request an invalid ID.